<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiView</class>
 <widget class="QWidget" name="MultiView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0" colspan="7">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="pageList">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QListView" name="listView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageTree">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTreeView" name="treeView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageTable">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>195</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButtonViewGrid">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/icons.qrc">
       <normaloff>:/icons/view_grid.png</normaloff>:/icons/view_grid.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupView</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButtonViewList">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/icons.qrc">
       <normaloff>:/icons/view_list.png</normaloff>:/icons/view_list.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupView</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButtonViewDetails">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/icons.qrc">
       <normaloff>:/icons/view_details.png</normaloff>:/icons/view_details.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupView</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="pushButtonViewTree">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/icons.qrc">
       <normaloff>:/icons/view_tree.png</normaloff>:/icons/view_tree.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupView</string>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupView"/>
 </buttongroups>
</ui>
